LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Apache 2.0 is better ;)   We've moved away from BSD to Apache 2.0 because it includes (among other things) a patent/retaliation [clause](http://en.swpat.org/wiki/Patent_clauses_in_software_licences#Apache_License_2.0).